Index: resources/views/index.blade.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- resources/views/index.blade.php	(date 1534886235000)
+++ resources/views/index.blade.php	(date 1534886312000)
@@ -126,6 +126,7 @@
                     <p>Gérant de cuisine et des services</p>
                 </div>
                 <div class="col-2">
+                    <!-- shadow comment -->
                     <img src="{{asset("images/lamjad.jpg")}}" class="img-fluid img-equipe"/>
                 </div>
                 <div class="col-2">
